engineering/conventions/go.md

```diff
@@ -7,17 +7,17 @@ This guide documents development conventions for Go at source{d}. Check [general
 
 ## Supported Go Versions
 
-* Our libraries support latest two stable major versions of Go (e.g. 1.10.x and 1.11.x). Both versions should be included in Travis CI.
+* Our **libraries** support latest two stable major versions of Go (1.11.x and 1.12.x). Both versions should be included in Travis CI.
   * Some of our libraries may support more versions, specially when there is wide-adoption outside source{d}, (e.g. [go-git](https://github.com/src-d/go-git) which currently supports three.
-* Our applications support only latest [stable](https://golang.org/dl/#stable) Go version.
+* Our **applications** support only latest [stable](https://golang.org/dl/#stable) Go version (1.12.x).
 
 ## Dependency Management
 
-* If your project is an application:
-  * Use [dep](https://github.com/golang/dep) to manage dependencies.
-  * Check in the `vendor` directory into git.
-* If your project is a library:
+* If your project is a **library**:
   * Use [gopkg.in](http://labix.org/gopkg.in) to provide versioned imports.
+* If your project is an **application**:
+  * Use [modules](https://github.com/golang/go/wiki/Modules) to manage dependencies.
+  * Check in the `vendor` directory into git (with `go mod vendor`).
```

Comment:
We should probably drop this requirement for applications.

Reply:
Sure, I also think it's not needed anymore. But wanted to discuss it separately, because it may touch other things like modules proxy, etc.

But I can make this change right now if @smola and @mcuadros are okay with it.

Reply:
Discussing it separately makes sense—but I wanted to raise the question.